pwiz_tools/Skyline/TestPerf/DiaUmpireTutorialTest.cs

```diff
@@ -0,0 +1,922 @@
+                ScoringModelCoefficients = "0.1684|-0.8487|3.8097|1.1177|-0.0577|0.7095|0.1366|-0.0444",
+                MassErrorStats = new[]
+                {
+                    new[] {1.8, 4},
+                    new[] {1.3, 3.9},
+                    new[] {1.9, 4.1},
+                    new[] {2, 3.8},
+                    new[] {2, 4.1},
+                    new[] {2.1, 3.9},
+                    new[] {1.8, 4.1},
+                },
+            };
+
+            if (!IsCoverShotMode)
+                TestQeData();
+        }
+       
+        private void TestQeData()
+        {
+            SetInstrumentType(new InstrumentSpecificValues
+            {
+                InstrumentTypeName = "QE",
+                DiaFiles = new[]
+                {
+                    "collinsb_X1803_171-A.mzML",
+                    "collinsb_X1803_172-B.mzML",
+                    "collinsb_X1803_173-A.mzML",
+                    "collinsb_X1803_174-B.mzML",
+                    "collinsb_X1803_175-A.mzML",
+                    "collinsb_X1803_176-B.mzML",
+                },
+                HasAmbiguousMatches = false,
+                IsolationSchemeName = "ETH QE (18 variable)",
+                IsolationSchemeFile = "QE_DIA_18var.tsv",
+                IsolationSchemeFileSeparator = TextUtil.SEPARATOR_TSV,
+                PrecursorTolerance = new MzTolerance(10, MzTolerance.Units.ppm),
+                FragmentTolerance = new MzTolerance(20, MzTolerance.Units.ppm),
+                InstrumentPreset = DiaUmpire.Config.InstrumentPreset.QExactive
+            });
+
+            RunTest();
+        }
+
+        // disable audit log comparison for FullFileset tests
+        public override bool AuditLogCompareLogs => !TestContext.TestName.EndsWith("FullFileset");
+
+        private void SetInstrumentType(InstrumentSpecificValues instrumentValues)
+        {
+            _instrumentValues = instrumentValues;
+
+            RootName = "DIA-" + InstrumentTypeName;
+            // LinkPdf = "https://skyline.ms/_webdav/home/software/Skyline/%40files/tutorials/{0}-20_1.pdf";
+            LinkPdf = string.Format("file:///C:/proj/branches/work/pwiz_tools/Skyline/Documentation/Tutorials/{0}-20_1.pdf", RootName);
+
+            TestFilesZipPaths = new[]
+            {
+                string.Format(@"http://skyline.ms/tutorials/{0}.zip", RootName),
+                string.Format(@"TestPerf\DiaSwath{0}Views.zip", InstrumentTypeName)
+            };
+
+            TestFilesPersistent = new[] { Path.Combine(RootName, "DDA_search"), Path.Combine(RootName, "DIA") };
+        }
+
+        private void RunTest()
+        {
+//            IsPauseForScreenShots = true;
+//            RunPerfTests = true;
+//            IsPauseForCoverShot = true;
+            CoverShotName = "DIA-SWATH with DiaUmpire";
+
+            RunFunctionalTest();
+
+            Assert.IsFalse(IsRecordMode);   // Make sure this doesn't get committed as true
```

Comment:
This assert here fails if you check "Record Audit Logs" in SkylineTester (and the test will not get run in subsequent languages).

If you want to keep this assert here, I think you need to change "IsRecordMode" so that it does something other than return "IsRecordAuditLogForTutorials".